client/css/jsonedit.css

```diff
@@ -426,6 +426,12 @@ body.jsonEdit.jeZoomOut #topSurface {
   opacity: 0.5;
 }
 
+/* For the zoom, edit, wide, highlight, and debug "on" states */
+[id=jeTopButtons] button.onState {
```

Comment:
Is `[id=jeTopButtons]` being used instead of `#jeTopButtons` to reduce selector specificity here?

Reply:
I was mirroring the syntax slightly further up in that file. Is there a difference between `[id=jeTopButtons]` and `#jeTopButtons`?

Reply:
> I was mirroring the syntax slightly further up in that file. Is there a difference between `[id=jeTopButtons]` and `#jeTopButtons`?

CSS rules are applied according to [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). More specific rules trump less specific rules regardless of where the rules appear. Although `#jeTopButtons` and `[id=jeTopButtons]` select the same element, `#jeTopButton` is considered more specific because `[id=jeTopButtons]` is evaluated as an attribute selector rather than the special id selector (same weight as a class selector).

Reply:
The only use of `#jeTopButtons` as a selector appears directly above the code you commented on. I don't see why replacing all occurrences of `[id=jeTopButtons]` with `#jeTopButtons` would cause any issues, but all of that was there before I started working on this, and I've been afraid to change it given my level of understanding of CSS.

Reply:
I suppose using attribute selectors to select an ancestor of the target widget by id is simply best practice because `#jeTopButtons button.onState` would higher calculated specificity (1-1-1) than `#je_toggleZoom.onState` (1-1-0) even though `#je_toggleZoom.onState` is actually more specific.